src/Address/AddressInterface.php

```diff
@@ -39,28 +39,28 @@ public function getBytes();
     /**
      * Get the type of the IP address.
      *
-     * @return int One of the \IPLib\Address\Type::T_... constants
+     * @return int One of the \IPLib\Address\Type constants
      */
     public function getAddressType();
 
     /**
      * Get the default RFC reserved range type.
      *
-     * @return int One of the \IPLib\Range\Type::T_... constants
+     * @return int One of the \IPLib\Range\Type constants
      */
     public static function getDefaultReservedRangeType();
 
     /**
      * Get the RFC reserved ranges (except the ones of type getDefaultReservedRangeType).
      *
-     * @return \IPLib\Address\AssignedRange[] ranges are sorted
+     * @return AssignedRange[] ranges are sorted
```

Comment:
In PHPDoc comments I do prefer fully-qualified class names.

Why?

Because:

1. You may need a class in a phpdoc that's not in a `use` statement because the code doesn't use it. So you may end up with having fully-qualified and non-fully-qualitied class names in phpdoc. Sure, you could add such classes to phpdoc, but I do prefer to avoid it if the classes are not used in the code.
2. You may "rename" used classes in `use` statements. For example, you may have "use Library\ClassA as ClassB"; in your code you should then write `@return ClassB`, but the reader would then need to read the `use` statements to see what's this `ClassB`.

That's why I do prefer writing fully qualified class names in PHPDoc.

Reply:
Right. I get your point, thanks.